addons/mail/static/src/model/model_manager.js

```diff
@@ -917,6 +917,46 @@ class ModelManager {
         }
     }
 
+    /**
+     * Executes the provided function as part of a single update cycle. This
+     * allows the execution of computed fields to happen only once, at the end
+     * of the last pending update cycle.
+     *
+     * @private
+     * @param {function} func
```

Comment:
would clarify this function is intended to make changes by mean of `.update()`, just so it's easier to understand why it makes sense to wrap this function.